examples/data/cleansed/metadata.yml

```diff
@@ -0,0 +1,61 @@
+scada:
+  time: time
+  id: id
```

Comment:
It looks like -25 has a tag name WTUR.TurNam for turbine name. But I'm happy to stay with "id"

Reply:
Updated.